src/main/java/de/themorpheus/edu/taskservice/endpoint/dto/response/solution/GetAllUserImageSolutionResponseDTO.java

```diff
@@ -0,0 +1,17 @@
+package de.themorpheus.edu.taskservice.endpoint.dto.response.solution;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import javax.validation.constraints.Size;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class GetAllUserImageSolutionResponseDTO {
+
+	@Size(min = 1)
+	private List<GetAllUserImageSolutionsResponseDTOModel> url;
```

Comment:
rename to userSolutions

Reply:
Not good because it returns an url.
So i would make userSolutionUrls maybe.

Reply:
rename it userSolutions
bc at this point its not neccessary to know if it is a url or a solution
the list just includes the solutions that the user sent